tools/jenkins/genDllVersions.sh

```diff
@@ -317,13 +319,14 @@ catDll libpoppler-glib-
 catDll librevenge-0
 catDll librevenge-stream-
 catDll libzip
-catDll libmad-0
-catDll libsox-
+#catDll libmad-0
```

Comment:
These don't appear to be needed anymore, but I am not sure why. If you know what they were used for please let me know and I can try to investigate more.

Reply:
The audiocurve node uses libsox. libmad is required by libsox to read MP3's.